barretenberg/cpp/src/barretenberg/ultra_honk/protogalaxy.test.cpp

```diff
@@ -14,9 +14,11 @@ template <typename Flavor> class ProtoGalaxyTests : public testing::Test {
   public:
     using Composer = UltraComposer_<Flavor>;
     using VerificationKey = typename Flavor::VerificationKey;
-    using Instance = ProverInstance_<Flavor>;
-    using Instances = ProverInstances_<Flavor, 2>;
-    using ProtoGalaxyProver = ProtoGalaxyProver_<Instances>;
+    using ProverInstance = ProverInstance_<Flavor>;
```

Comment:
Diffs in this file only relate to separating the prover and verifier instances explicitly